pyproject.toml

```diff
@@ -67,7 +67,9 @@ dev = [
     "pre-commit",
     "pytest",
     "build",
-    "fuse-python ; platform_system == 'Linux'"
+    "fuse-python ; platform_system == 'Linux'",
+    "clip@git+https://github.com/openai/CLIP.git",
```

Comment:
since this is for a test, should we target more than just their default branch - it doesn't look like it has any releases, unfortunately so maybe a specific commit?

Reply:
pinned.